cruise-control/src/main/scala/com/linkedin/kafka/cruisecontrol/executor/ExecutorUtils.scala

```diff
@@ -26,16 +26,16 @@ object ExecutorUtils {
    * within the cluster, (2) introduce a new replica to the cluster (3) remove an existing replica from the cluster.
    *
    * @param kafkaZkClient the KafkaZkClient class to use for partition reassignment.
-   * @param reassignmentTasks Replica reassignment tasks to be executed.
+   * @param tasksToExecute Replica reassignment tasks to be executed.
    */
   def executeReplicaReassignmentTasks(kafkaZkClient: KafkaZkClient,
```

Comment:
After switch to use admin client, this method seems redundant. should remove this one?

Reply:
Marked as `deprecated` -- thanks!